src/extension/inspire/src/test/java/org/geoserver/inspire/wms/WMSExtendedCapabilitiesTest.java

```diff
@@ -228,6 +229,24 @@ public void testResponseLanguageIsDefault2() throws Exception {
             if (isLangNode(el)) language = el.getTextContent();
         }
         assertEquals("fre", language);
+
+        String responseString = getAsServletResponse(WMS_1_3_0_GETCAPREQUEST).getContentAsString();
+        assertTrue(
```

Comment:
Testing is weak. Please navigate the dom as done above, or use XMLAssert to perform xpath assertions.

Reply:
done